wakatime-mode.el

```diff
@@ -196,6 +195,10 @@
                  (wakatime-call ,savep t)
                )
              )
+             ; finally throw unexpected errors
```

Comment:
Isn't that what happens here? Oh does the error throwing not exit the function? My bad